main/src/library/Prelude.flix

```diff
@@ -129,6 +129,17 @@ pub def println(x: a): Unit \ IO with ToString[a] =
     import java.io.PrintStream.println(String): Unit \ IO;
     x |> ToString.toString |> println(getOut())
 
+///
+/// Prints the given string `x`.
+///
+/// Note: For internal use only. The actual `debug(exp)` expression is parsed and ultimately rewritten to call this function.
+///
+@Internal
+def debug(x: String): Unit \ IO =
+    import static get java.lang.System.out: ##java.io.PrintStream \ IO as getOut;
```

Comment:
Can we instead print to StdErr?

Reply:
Not sure I like that. Whats the pros and cons?

Reply:
Pros: 
1. We don't interfere with StdOut. If I have a Flix program as part of a pipeline and I'm debugging, I don't want debugging information going out with the actual output.
2. Principle: debugging information is not generally primary output from a program

Cons:
1. Depending on where things are flushed, might not print to output when we expect, with respect to other println calls

Incidentally, we should probably add a flush so this occurs right away instead of hanging out in the buffer.

Reply:
I believe println flushes.